api/src/pcapi/local_providers/cinema_providers/cds/cds_stocks.py

```diff
@@ -134,3 +207,19 @@ def get_next_product_id_from_database():  # type: ignore [no-untyped-def]
 def get_next_offer_id_from_database():  # type: ignore [no-untyped-def]
     sequence = Sequence("offer_id_seq")
     return db.session.execute(sequence)
+
+
+def _format_datetime_from_local_timezone_to_utc(date: datetime, local_tz: str) -> datetime:
```

Comment:
(détail / nommage) Si l'objet est un `datetime`, l'appeler "date" est un peu déroutant, parce que ça n'est pas une `date`. `dt` semble être un terme raisonnable et assez courant :

```suggestion
def _format_datetime_from_local_timezone_to_utc(dt: datetime, local_tz: str) -> datetime:
```

Reply:
done